### PR TITLE
rebuild 18.16 for icu 73 support

### PR DESCRIPTION
**Pull request by boldorider4**

### CODE REVIEW

**Comment by boldorider4** [comment]
when all looks good I will remove `abs.yaml`

File: recipe/build.sh
Comment (danpetry) [inline recipe/build.sh:30]: just wondering if this is necesssary? Usually it isn't, and it builds without on my local machine.

File: recipe/meta.yaml
Comment (danpetry) [inline recipe/meta.yaml:51]: you need to update the lower bounds here as well, and the upper bound isn't correct. Have a look at what it was before, and how the host version related to the runtime version range. Also, try and figure what the explanation might be for that. (there's a comment in the icu recipe that relates to this. Also, it would be useful to read up on `run_exports` and `pin_subpackage` in the [conda-build docs](https://docs.conda.io/projects/conda-build/en/latest/index.html))

File: recipe/build.sh
Comment (boldorider4) [reply recipe/build.sh:30]: Interestingly enough, if I don't include it, every build breaks (also on Prefect) because `conda_build.sh` tries to execute `configure` from a path one level up with respect to `SRC_DIR`. If the fix needed to be somewhere else, I don't know, but this certainly fixes it.

File: recipe/build.sh
Comment (cbouss) [inline recipe/build.sh:38]:
```suggestion
```

We do want internationalization, it was removed before because our icu was too old.
The comment line 40/41 should be removed as well.

File: recipe/meta.yaml
Comment (cbouss) [inline recipe/meta.yaml:52]:
```suggestion
    - {{ pin_compatible('icu') }}
```
This should replace line 51/52. This results in icu >=73.1,<74.0a0

**Review by cbouss — COMMENTED** [review]
To coordinate with the openssl release. See https://github.com/AnacondaRecipes/nodejs-feedstock/pull/15

**Review by danpetry — APPROVED** [review]

**Review by cbouss — APPROVED** [review]
LGTM, but this should merged after #15 is merged

**Review by JeanChristopheMorinPerso — APPROVED** [review]
Like @cbouss said, LGTM and it should go after the 18.16 update with OpenSSL 3. THis will require a rebase (or a merge).

**Comment by M-Waszkiewicz-Anaconda** [comment]
> Like @cbouss said, LGTM and it should go after the 18.16 update with OpenSSL 3. THis will require a rebase (or a merge).

yes, I will rebase it later, just ping me out once it is ready

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
this is temporary only as you described earlier LGTM

**Pull request merged**
